### PR TITLE
back button correctly aligned in RTL (bug 1179168)

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
padding added in RTL

### CODE REVIEW

**Comment by mstriemer** [comment]
I'm not sure if this should be on the left or right. Confirming with @pwalm.

File: src/media/css/elements--nav.styl [hunk 1]
```diff
@@ -212,7 +212,10 @@ mkt-nav-child {
     transition: unquote($nav--transition);
     width: $nav--width;
     z-index: $zindex--nav-hamburger-child;
-
+    
+    +rtl() {
+        padding: 0 22px 0 0;
```
Comment (mstriemer) [inline src/media/css/elements--nav.styl]: Can you update the other padding rule to be `0 22px` instead of overriding for RTL please.

**Comment by mstriemer** [comment]
This is the right change, btw. So I'll merge it after the `padding` rules are combined. Thanks.

**Comment by atiqueahmedziad** [comment]
done. Thanks :)

**Comment by mstriemer** [comment]
Thanks Atique!

**Pull request merged**